### PR TITLE
Change SDK version to 7.0.0

### PR DESCRIPTION
**Pull request by ninovanhooff**
CURA-6858

Companion PRs

https://github.com/Ultimaker/Uranium/pull/534
https://github.com/Ultimaker/CuraPrintProfileCreator/pull/36

### CODE REVIEW

File: plugins/AMFReader/plugin.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
     "author": "fieldOfView",
     "version": "1.0.0",
     "description": "Provides support for reading AMF files.",
-    "api": "6.0.0"
+    "api": "7.0.0"
```
Comment (LipuFei) [inline plugins/AMFReader/plugin.json]: To keep it consistent with the others, I'd change this to `7.0`

File: plugins/TrimeshReader/plugin.json [hunk 2]
```diff
@@ -3,5 +3,5 @@
     "author": "Ultimaker B.V.",
     "version": "1.0.0",
     "description": "Provides support for reading model files.",
-    "api": "6.0.0"
+    "api": "7.0.0"
```
Comment (LipuFei) [inline plugins/TrimeshReader/plugin.json]: To keep it consistent with the others, I'd change this to `7.0`

File: plugins/UFPReader/plugin.json [hunk 3]
```diff
@@ -3,6 +3,6 @@
     "author": "Ultimaker B.V.",
     "version": "1.0.0",
     "description": "Provides support for reading Ultimaker Format Packages.",
-    "supported_sdk_versions": ["6.0.0"],
+    "supported_sdk_versions": ["7.0.0"],
```
Comment (LipuFei) [inline plugins/UFPReader/plugin.json]: To keep it consistent with the others, I'd change this to `7.0`

File: plugins/AMFReader/plugin.json (hunk 1 above)
Comment (nallath) [reply plugins/AMFReader/plugin.json]: It must be a semver, so it's better to have it like 7.0.0

File: plugins/TrimeshReader/plugin.json (hunk 2 above)
Comment (nallath) [reply plugins/TrimeshReader/plugin.json]: Same as above, 7.0.0 is correct semver, 7.0 (even though we accept it) is technically not.

File: plugins/UFPReader/plugin.json (hunk 3 above)
Comment (nallath) [reply plugins/UFPReader/plugin.json]: Same as above, 7.0.0 is correct semver, 7.0 (even though we accept it) is technically not.

**Pull request merged**